app/minutes/test_minutes.py

```diff
@@ -289,7 +289,7 @@ def test_create_minute_normal_member_private(self):
         self.socketio.emit("create_minute", self.user_data)
         received = self.socketio.get_received()
         response = received[0]["args"][0]
-        assert response == Response.PermError
+        assert response == Response.AddMinuteSuccess
```

Comment:
Normal members are now allowed to create private minutes, so this no longer causes a permission error.